cli/ballerina-tool/src/main/resources/cli-help/ballerina-update.help

```diff
@@ -5,10 +5,10 @@ SYNOPSIS
     ballerina update
 
 DESCRIPTION
-     Ballerina update command will update to provided version.
+     The `ballerina update` command will update the Ballerina SDK version to the provided one.
```

Comment:
It says the provided one. But according to the synopsis, the command doesn't seem to accept any arguments.

Reply:
Changed as follows:

"The `ballerina update` command will update the Ballerina distribution version to the latest."